versions.json

```diff
@@ -195,15 +195,15 @@
         "build_steps": [
```

Comment:
also set:

"consortium_build_number": "1",

and then we can squash and #

Reply:
ok, will do, once testing is complete